.gitmodules

```diff
@@ -37,3 +37,6 @@
 [submodule "verif/core-v-verif"]
 	path = verif/core-v-verif
 	url = https://github.com/openhwgroup/core-v-verif
+[submodule "core/cache_subsystem/hpdcache"]
```

Comment:
Can you use https, ot be aligned on others

Reply:
Yes, I can do that